unity/Assets/Scripts/ImageSynthesis/Shaders/Depth.shader

```diff
@@ -20,6 +22,9 @@ Shader "Hidden/Depth" {
              uniform sampler2D _CameraDepthTexture;
              uniform fixed _DepthLevel;
              uniform half4 _MainTex_TexelSize;
+
+             float _NearClipPlane;
+             float _FarClipPlane;
```

Comment:
This uniforms are not needed as unity provides the builtin variable `_ProjectionParams`, where `_ProjectionParams.y` is the near plane and `_ProjectionParams.z` is the far plane. https://docs.unity3d.com/Manual/SL-UnityShaderVariables.html

Reply:
Nice! I'll switch to using those.